src/components/Layout/Layout.js

```diff
@@ -49,7 +53,23 @@ const Layout = ({ children }) => {
         <PrimaryNav />
       </div>
       <main className="l-main" id="main-content">
-        {children}
+        <div data-test="main-children">{children}</div>
+        {!cookies.release20_04 && (
+          <Notification
+            type="information"
+            close={() => {
+              setCookies("release20_04", true);
+            }}
+          >
+            This is the new JAAS Dashboard, replacing the previous Juju GUI here
```

Comment:
```suggestion
            Thanks for trying the new JAAS Dashboard! This dashboard is replacing the Juju GUI
```

Reply:
Text as per @clagom's instruction https://codepen.io/clagom/pen/LYpxYMe?editable=true%3Dhttps%3A%2F%2Fcodepen.io%2Fembed%2Fprefill%2F

Reply:
Well that instruction is incorrect 😆  At the very least we can't say "here on jaas.ai" as this message will also be shown when it's not served from jaas.ai :) (it's also a comma splice ;) )